proxy/src/graphql/schema.rs

```diff
@@ -366,6 +381,24 @@ impl ControlMutation {
 
         Ok(true)
     }
+
+    fn register_user(
+        ctx: &Context,
+        handle: juniper::ID,
+        id: juniper::ID,
+    ) -> Result<registry::Transaction, error::Error> {
+        // TODO(xla): Get keypair from persistent storage.
+        let fake_pair = ed25519::Pair::from_legacy_string("//Alice", None);
```

Comment:
You could make this function receive the seed for the ed25519 pair instead of hardcoding it to Alice.

Reply:
Then we also need to ensure that that account has funds, will leave it until the needs arises.